### ISSUE TITLE
Add fromString method to ContractId

### ISSUE THREAD

**Issue by cylon56** [hashgraph/hedera-sdk-java#163]
Right now, there isn't a way to create a new ContractId object from a key String. Having a fromString method would be incredibly useful for Hedera applications that require a user to input a contract ID among other uses. See the example below:
```
System.out.println("Enter the contract ID:");
String contractIdString = s.nextLine();
System.out.println("");
ContractId  newContractId = new ContractId.fromString(contractIdString);
```
I'd be willing to work on this feature upon acceptance.

**Comment by lbaird** [comment]
That’s a great idea. It should work as a round trip, so this is always true:

x == ContractId.fromString(x.toString())

It should also be more general in what it accepts. A number with leading zeros should be accepted. A separator of comma or space should be accepted. It might be best to say it will accept any string that contains exactly 3 substrings of contiguous digits, and allows leading zeros. So “1.2.3” would be the same as “$)hs dfgc 001 ^} }]2/3–&(/sh”.

A regular expression would be an easy way to extract the 3 sub strings. And the Long.fromString could be applied to each.


________________________________

**Comment by cylon56** [comment]
Sounds good. I'll start work this weekend.

**Comment by mehcode** [comment]
https://github.com/hashgraph/hedera-sdk-java/blob/master/src/main/java/com/hedera/hashgraph/sdk/account/AccountId.java#L27

It would be nice to think of a way to avoid that function in 3 places (Account, Contract, and File).

**Comment by lbaird** [comment]
Maybe we could have 3 separate methods, but each of them calls the same static utility method, which converts a string to an array of 3 longs.


________________________________

**Comment by abonander** [comment]
This should be closed now that #164 is merged.

**Comment by abonander** [comment]
This should be available in the next release of the SDK which we just published today.
